### PR TITLE
Master for docker 1

### PR DESCRIPTION
**Pull request by gbasolu**
Added some things needed for docker

### CODE REVIEW

File: hydra_base/hydra.ini [hunk 1]
```diff
@@ -19,10 +19,16 @@ compression_threshold=50000
 
 [mysqld]
 user = root
-password = root
```
Comment (knoxsp) [inline hydra_base/hydra.ini]: Please don't change config stuff in the repo unless it's new or for a very good reason.

File: hydra_base/hydra.ini [hunk 2]
```diff
@@ -43,6 +51,8 @@ layout_xsd_path   = %(hydra_base_dir)s/static/resource_layout.xsd
 [hydra_client]
 #url = http://ec2-54-229-95-247.eu-west-1.compute.amazonaws.com/hydra-server?wsdl
 domain = http://127.0.0.1
+# Hydra server
+# domain = 172.20.0.3
```
Comment (knoxsp) [inline hydra_base/hydra.ini]: Remove commented lines please

File: hydra_base/util/hdb.py [hunk 3]
```diff
@@ -107,9 +107,9 @@ def make_root_user():
 
 
     db.DBSession.flush()
-
```
Comment (knoxsp) [inline hydra_base/util/hdb.py]: This file has no changes. Don't include in merge request please

Comment (gbasolu) [reply hydra_base/util/hdb.py]: Atom deleted the tabs/spaces on empty lines and then finished into the diffs

File: hydra_base/hydra.ini (hunk 1 above)
Comment (gbasolu) [reply hydra_base/hydra.ini]: Restored them

File: hydra_base/hydra.ini (hunk 2 above)
Comment (gbasolu) [reply hydra_base/hydra.ini]: Done it

**Pull request merged**